### ISSUE TITLE
Abnormalities of alveolar macrophages

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5556]
**Preferred term label:**
Abnormalities of alveolar macrophages

**Synonyms**

**Definition (free text, please give PubMed ID)**
Alveolar macrophages usually make up the majority of cells in the bronchoalveolar space (> 80%). The may contain intracellular material depending on underlying diseases or due to exposition to inhaled particles.
Costabel U. Atlas of bronchoalveolar lavage. London: Chapman & Hall; 1998. ISBN0412792702.

**Parent term (use hpo.jax.org/app)**
Abnormal cellular composition of broncho-alveolar fluid 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
@GrieseAG
@Katrinknoflach

**Comment by pnrobinson** [comment]
added term
